src/Diff.jl

```diff
@@ -1,35 +1,38 @@
-export Rotor, generator, Diff, backward!, classical_autodiff!, CPhaseGate, DiffBlock
+"""
+Quantum Differentiable Blocks.
+"""
+
+export generator, Diff, CPhaseGate
 export dispatch_to_diff!, parameters_of_diff
 import Yao: expect, content, chcontent, mat, apply!
 using StatsBase
 
+using Yao
```

Comment:
it is better to remove the dependency of Yao since this is going to be inside it.